pkg/cmd/roachtest/tests/kv.go

```diff
@@ -170,9 +167,14 @@ func registerKV(r registry.Registry) {
 				sequential = " --sequential"
 			}
 
-			var envFlags string
-			if e := os.Getenv(envKVFlags); e != "" {
-				envFlags = " " + e
+			var duration string
+			envWorkloadDurationFlag := os.Getenv(test.EnvWorkloadDurationFlag)
+			defaultDuration := ifLocal(c, "10s", opts.duration.String())
```

Comment:
Nit: I'd swap the order so that `envWorkloadDurationFlag` logic is on consecutive lines.

Reply:
Updated